### ISSUE TITLE
Linked filter and filter type: multiple

### ISSUE THREAD

**Issue by Nexpola** [koalyptus/TableFilter#356]
Hi,
I'm having problems when using linked filters and multiple filter.
When i select the first item, menu list automatically update itself and I can't select a second option even if I keep CTRL pressed.
It could be possible to avoid dropdown list autorefresh when CTRL is keep pressed or until menu lost its focus/disappear?

Thank you!

**Comment by koalyptus** [comment]
Hi @Nexpola,
In your scenario, to have multiple selection, you can still hold the mouse button pressed and choose contiguous options: 
![image](https://cloud.githubusercontent.com/assets/8214629/20854240/31c96b66-
b946-11e6-8689-0222df46be5c.png)

This is a good enhancement candidate , a nice to have, but not super urgent.
Cheers

**Comment by Nexpola** [comment]
Yes, holding the mouse button and dragging is the only solution I've found.  
But it's not so easy to use when dropdown list is quite long and I have to select only few options... (the first one on top and the last one at the bottom of the list, as always :sweat_smile: )

Thank you!  :relaxed:

**Comment by koalyptus** [comment]
I know, not ideal, needs definitely to be improved.
